src/pages/academy/components/_hero.js

```diff
@@ -71,27 +76,34 @@ const Hero = ({ imageAlt, imageData, title, description }) => {
                         text_align={'left'}
                         font_size={['64px', '32px']}
                         line_height={['80px', '40px']}
-                        max_width={['785px', '237px']}
+                        max_width={['636px', '328px']}
                     >
                         {title}
                     </Title>
                     <TextWrapper
                         color={'white'}
                         font_size={['20px', '14px']}
                         line_height={['30px', '24px']}
-                        max_width={['684px', '328px']}
+                        max_width={['636px', '328px']}
                     >
                         {description}
                     </TextWrapper>
+                    <Flex jc="flex-start" height="unset" mt="24px" tabletL={{ mt: '16px' }}>
+                        <LinkButton id="hero-article" to={href} width="auto" hero>
+                            {cta_link}
```

Comment:
```suggestion
                            {button_text}
```
this would be more descriptive.